app/dose-series-service/dose-series.service.ts

```diff
@@ -0,0 +1,143 @@
+import {Injectable} from "@angular/core";
+import {AuthHttp} from "../angular2-jwt";
+import {BACKEND_URL} from "../backend-config/backend-config";
+import {DoseSeries} from "../dose-series/dose-series";
+import {RequestOptions, Headers} from "@angular/http";
+
+/**
+ * Service for fetching and saving DoseEvent instances.
+ */
+@Injectable()
+export class DoseSeriesService {
```

Comment:
I created this new `DoseSeriesService` based on the REST endpoint documented on the wiki.

It sounds like the server didn't implement all of these endpoints though.

If we want users to be able to open a med and edit the med and schedule, we will need them though...